mizar/dp/mizar/workflows/builtins/networkpolicies/delete.py

```diff
@@ -35,9 +37,9 @@ def run(self):
         logger.info("Run {task}".format(task=self.__class__.__name__))
 
         policy_name = "{}:{}".format(self.param.namespace, self.param.name)
-        affected_endpoint_names = networkpolicy_util.update_and_retrieve_affected_endpoint_names(policy_name, None)
+        affected_endpoint_names = networkpolicy_util.update_and_retrieve_affected_endpoint_names(policy_name, None, None)
```

Comment:
Changed affected_endpoint_names to endpoint_names.
There is networkpolicy_endpoints_store in operator store which stores mapping of policy to endpoints.
Here the function named update_and_retrieve_endpoint_names. Its name means by input of policy's pod selector, it will update networkpolicy_endpoints_store and return updated endpoint name list for later usage.